src/search.c

```diff
@@ -58,7 +58,7 @@ struct bundle_result {
 
```

Comment:
In commit message: dinamically -> dynamically

Reply:
Ugh, fixed. Thanks.